tests/integrations/tests/test_bedrock.py

```
@@ -104,9 +228,8 @@ def convert_to_bedrock_messages(messages: List[Dict[str, Any]]) -> List[Dict[str
                         })
                     else:
                         # URL image - download and convert to bytes
-                        response = requests.get(url, timeout=20)
-                        response.raise_for_status()
-                        image_bytes = response.content
+                        with urllib.request.urlopen(url, timeout=20) as response:
```

Ruff (0.14.7)

Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)